src/Hng.Application/Features/Products/Dtos/ProductDto.cs

```diff
@@ -6,4 +6,4 @@ public class ProductDto
         public string name { get; set; }
         public string description { get; set; }
```

Comment:
Don't use camelCase for Properties. Use Normal PascalCasing, then apply `JsonPropertyName` attribute to but it in snake_case